.storybook/preview.tsx

```diff
@@ -142,6 +142,7 @@ addParameters({
           'Buttons',
           'Tables',
           'Forms',
+          'Sidebar Section',
```

Comment:
Would this make more sense under the `Views` section I wonder

Reply:
oh yeah makes sense, I can move it there!